src/db/migrations/20240601114803_add_first_broker_removal_email_sent.js

```diff
@@ -0,0 +1,23 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+/**
+ * @param { import("knex").Knex } knex
+ * @returns { Promise<void> }
+ */
+export function up (knex) {
+  return knex.schema.table("subscribers", table => {
+    table.boolean("first_broker_removal_email_sent").defaultTo(false);
```

Comment:
```suggestion
    table.boolean("first_broker_removal_email_sent").defaultTo(false);
    table.index("first_broker_removal_email_sent");
```

Since production has millions of rows on this table and you're querying it regularly below, I suspect you'll want an index on this. We could try to measure beforehand, but GCP has tools that make it pretty easy to see the cost of queries so we can double-check it that way.

Reply:
Added the index for `irst_broker_removal_email_sent` in [9dc7c02](https://github.com/mozilla/blurts-server/pull/4617/commits/9dc7c028a3dbae1f56e393381089969064684ad3).